scaffolder-templates/gitops/bootstrap/README.md

```diff
@@ -14,4 +15,12 @@ SSH_PRIVATE_KEY=$(oc get secrets -n orchestrator-gitops git-ssh-credentials -o j
 sed -i "s/__REPLACE_SSH_PRIVATE_KEY__/$SSH_PRIVATE_KEY/" ${{values.workflowId}}-argocd-repo.yaml
 
 kubectl apply -f .
-```
\ No newline at end of file
+```
+
+**Note:** If you're not logged into the repository, you need to provide a personal access token (PAT) as part of the clone URL.
+
+```
+git clone https://<PAT>@github.com/${{ values.orgName }}/${{ values.repoName }}.git
+```
+
+Replace `<PAT>` with your personal access token. Ensure the token has the necessary permissions to access the repository.
```

Comment:
pls add also a reference to https://docs.github.com/en/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors
since this issue can be solved by more than one method

Reply:
done